File: PHP/Processor.php
<?php

class Processor {
    // variabel untuk menampung nama prosesor dan harganya
    private $name; 
    private $price; 

    // constructor
    function __construct($name, $price) {
        $this->name = $name;
        $this->price = $price;
    }

    // setter and getter
    function setName($name) {
        $this->name = $name;
    }

    function getName() {
        return $this->name;
    }

    function setPrice($price) {
        $this->price = $price;
    }

    function getPrice() {
        return $this->price;
    }

    // method
    function printProcessor() {
        echo "Nama Processor : ". $this->getName(). "<br/>";
        echo "Harga Processor : Rp ". $this->getPrice(). "<br/>";
    }
}

?>

File: CPP/Processor.cpp
#include <iostream>
#include <string>

using namespace std;

class Processor
{
    private:
        // variabel untuk menampung nama prosesor dan harganya
        string name;    
        int price;      
    
    public:
        // constructor
        Processor() {}

        Processor(string name, int price)
        {
            this->name = name;
            this->price = price;
        }

        // setter and getter
        void setName(string name)
        {
            this->name = name;
        }

        string getName()
        {
            return this->name;
        }

        void setPrice(int price)
        {
            this->price = price;
        }

        int getPrice()
        {
            return this->price;
        }

        void printProcessor()
        {
            cout << "Nama Processor : " << this->getName() << endl;
            cout << "Harga Processor : Rp " << this->getPrice() << endl;
        }

        // destructor
        ~Processor() {}
};